config/settings/development.yml

```diff
@@ -4,3 +4,6 @@ token_encrypt:
   salt: 'dcVNcekAYNIFtggdJXQpSkWHBpBVsVsPiRzCFpWbZGLrQtHKSdMBKIWKmYYIZnkbXuMisFlSkHbUmIimKMbjFMVDpfzyIRLvmLYBUparuPAaxYjPzdmFfRGBGgxfvwOR'
 
 mailer_host: 'localhost:3000'
+
+fake_work_groups:
+  cabeer: 'site-adminstrators'
```

Comment:
I think you may have not meant to check this in? (as well as https://github.com/sul-dlss/sul-requests/pull/148/files#diff-646ad6b10917e4385cbcc8c8524e24a2R2)